srcpkgs/graphviz/patches/cross.patch

```diff
@@ -0,0 +1,12 @@
+--- lib/gvpr/Makefile.am.orig
++++ lib/gvpr/Makefile.am
+@@ -52,8 +52,7 @@ libgvpr_la_DEPENDENCIES = $(libgvpr_C_la
+ 
+ mkdefs_SOURCES = mkdefs.c
+ 
+-gdefs.h: $(top_srcdir)/lib/gvpr/gprdata mkdefs$(EXEEXT)
+-	./mkdefs$(EXEEXT) gdefs.h < $(top_srcdir)/lib/gvpr/gprdata
```

Comment:
It should be possible to add `build_helper="qemu"` to the template and patch this line to
```
        qemu-${XBPS_TARGET_QEMU_MACHINE}-static ./mkdefs$(EXEEXT) gdefs.h < $(top_srcdir)/lib/gvpr/gprdata
```
in e.g. `pre_configure()` and inside a block
```
        if [ "$CROSS_BUILD" ]; then
                vsed -i lib/gvpr/Makefile.am -e 's;\./mkdefs;qemu-${XBPS_TARGET_QEMU_MACHINE}-static &;'
        fi
```
Then cross builds will run `mkdefs` under `qemu`.